src/overlay/OverlayManagerImpl.cpp

```diff
@@ -142,7 +146,8 @@ OverlayManagerImpl::PeersList::moveToAuthenticated(Peer::pointer peer)
     {
         CLOG(WARNING, "Overlay")
             << "Trying to move non-pending " << mDirectionString << " peer "
-            << peer->toString() << " to authenticated list";
+            << peer->toString() << " to authenticated list"
```

Comment:
I think you meant to have "to authenticated list" after the port. In order to avoid issues like this, I think it would be better to use a function like `std::string Peer::toStringWithPort() const`.

Reply:
no, the port is not related to `peer` but to the application, and I'd rather keep this at the end of the line to limit the confusion